analyses/focal-cn-file-preparation/README.md

```diff
@@ -23,6 +23,7 @@ Running the following from the root directory of the repository will run the ste
 ```
 RUN_ORIGINAL=1 bash analyses/focal-cn-file-preparation/run-prepare-cn.sh
 ```
+**Temporary Note**: The `run-prepare-cn.sh` script currently downloads the `gencode.v38.annotation.gtf.gz` file since it is not available in the data directory. When it is added to the set of files within the data directory, this script will need to be updated to remove the download requirement and instead read it along with the rest of the files.
```

Comment:
This will be available in data-release.

Reply:
@runjin326 I agree. Once the files for v11 are all ready, I can remove the download lines from this script and open another PR.